tools/ci/testlist/arm-13.dat

```diff
@@ -9,3 +9,5 @@
 -launchxl-tms57004:nsh
 -lm3s6965-ek:qemu-nxflat
 -tms570ls31x-usb-kit:nsh
+-tlsr8278adk80d:ostest
+-tlsr8278adk80d:nsh
```

Comment:
let's change to:
```
-tlsr8278adk80d
```
to filter out all config
BTW, please append to PR https://github.com/apache/incubator-nuttx/pull/6179, so we can verify it really work before merge.

Reply:
Done